src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -21,6 +21,7 @@
                 :align-items      :center
                 :justify-content  :center
                 :border-radius    (/ container-size 2)
-                :background-color (colors/custom-color-by-theme color 50 60)}
+                ;:background-color (colors/custom-color-by-theme color 50 60)
```

Comment:
could you elaborate?

Reply:
Commenting out without todo doesn't look much good. But this was done due to this: https://github.com/status-im/status-mobile/pull/14494#discussion_r1050603672

Maybe there is a better way?

Reply:
@flexsurfer I had the todo in the quo_preview file. I added it to this file as well.